packages/web/src/pages/dashboard.js

```diff
@@ -1,7 +1,21 @@
+//import modules
+import { useRouter } from 'next/router'
+
+import { Button } from 'components/Button'
+
 export default function DashboardPage() {
+	const router = useRouter();
 	return (
 		<div>
-			Dashboard
+			<div>
+				Dashboard
+			</div>
+			<div>
+			<Button onClick={() => {router.push('/campaign')}}>
```

Comment:
If we have a button that does nothing more than take you to another page, we should convert it to an anchor. You can use the [`next/link` component](https://nextjs.org/docs/api-reference/next/link).